### PR TITLE
feat(JMX): adapt JMX view from 2.7

### PR DESCRIPTION
**Pull request by SteKoe**

### CODE REVIEW

File: spring-boot-admin-samples/spring-boot-admin-sample-servlet/src/main/java/de/codecentric/boot/admin/SecurityPermitAllConfig.java [hunk 1]
```diff
@@ -40,11 +40,11 @@ public SecurityPermitAllConfig(AdminServerProperties adminServer) {
...
-				.csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse()).ignoringRequestMatchers(
-						new AntPathRequestMatcher(this.adminServer.path("/instances"), HttpMethod.POST.toString()),
-						new AntPathRequestMatcher(this.adminServer.path("/instances/*"), HttpMethod.DELETE.toString()),
-						new AntPathRequestMatcher(this.adminServer.path("/actuator/**"))));
-
+			.csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse()).ignoringRequestMatchers(
+				new AntPathRequestMatcher(this.adminServer.path("/instances"), HttpMethod.POST.toString()),
+				new AntPathRequestMatcher(this.adminServer.path("/instances/*"), HttpMethod.DELETE.toString()),
+				new AntPathRequestMatcher(this.adminServer.path("/actuator/**"))));
+		http.csrf().disable();
```
Comment (erikpetzold) [inline spring-boot-admin-samples/spring-boot-admin-sample-servlet/src/main/java/de/codecentric/boot/admin/SecurityPermitAllConfig.java:47]: configuring csrf token repository and then disabling csrf sounds weird

File: spring-boot-admin-server-ui/src/main/frontend/components/sba-input.vue [hunk 2]
```diff
@@ -139,7 +151,7 @@ export default {
 
       const classNames = [];
 
-      if (this.error !== undefined) {
+      if (!!this.error) {
```
Comment (erikpetzold) [inline spring-boot-admin-server-ui/src/main/frontend/components/sba-input.vue:154]: Is this some fancy JS syntax I don't know? Or should we remove the double negation?

File: spring-boot-admin-server-ui/package.json [hunk 3]
```diff
@@ -72,23 +72,24 @@
...
     "babel-loader": "8.3.0",
     "eslint": "7.32.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-storybook": "0.6.10",
     "eslint-plugin-vue": "^9.0.0",
-    "jest": "26.6.3",
-    "jest-environment-jsdom": "27.5.1",
+    "jest": "29.3.1",
+    "jest-environment-jsdom": "29.3.1",
```
Comment (erikpetzold) [inline spring-boot-admin-server-ui/package.json:85]: updating jest could have been a separate PR ;)

File: spring-boot-admin-server-ui/src/main/frontend/components/sba-input.vue (hunk 2 above)
Comment (SteKoe) [reply spring-boot-admin-server-ui/src/main/frontend/components/sba-input.vue]: Nah, this is not fancy. I have replaced it by an explicit check of null or undefined :)

File: spring-boot-admin-server-ui/package.json (hunk 3 above)
Comment (SteKoe) [reply spring-boot-admin-server-ui/package.json]: Right :)

**Pull request merged**